lib/SILOptimizer/Transforms/RedundantLoadElimination.cpp

```diff
@@ -160,6 +160,9 @@ static bool isRLEInertInstruction(SILInstruction *Inst) {
   case SILInstructionKind::IsEscapingClosureInst:
   case SILInstructionKind::IsUniqueInst:
   case SILInstructionKind::FixLifetimeInst:
+  case SILInstructionKind::EndAccessInst:
+  case SILInstructionKind::SetDeallocatingInst:
+  case SILInstructionKind::DeallocRefInst:
```

Comment:
I wonder if we can centralize this notion in a helper. Just a random thought.

Reply:
There is _a lot_ that could be centralized between DSE and RLE. They both end up doing the work of the other pass anyway so, it might make sense to combine them.